server/routes/api/v1/v1.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express')
+const routeCollector = express.Router() 
+
+// Require the actual route files
+const test = require('../../../__tests__/unit/post_admin_test');
```

Comment:
We should not need to require any tests in this way. Can you explain the purpose of creating a test endpoint here?